Tests/appstoreconnect-cliTests/Operations/ModifyBetaGroupOperationTests.swift

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+@testable import AppStoreConnectCLI
+@testable import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+import XCTest
+
+final class ModifyBetaGroupOperationTests: XCTestCase {
+    typealias Operation = ModifyBetaGroupOperation
+    typealias Options = Operation.Options
+    typealias App = AppStoreConnect_Swift_SDK.App
+    typealias BetaGroup = AppStoreConnect_Swift_SDK.BetaGroup
+
+    let testUrl = URL(fileURLWithPath: "test")
+
+    var app: App {
+        App(
+            attributes: App.Attributes(
+                bundleId: "com.example.test",
+                name: nil,
+                primaryLocale: nil,
+                sku: nil
+            ),
+            id: "1234567890",
+            relationships: nil,
+            links: ResourceLinks(self: testUrl)
+        )
+    }
+
+    var options: Options {
+        Options(
+            app: app,
+            currentGroupName: "Some Group",
+            newGroupName: "New Group Name",
+            publicLinkEnabled: true,
+            publicLinkLimit: 10,
+            publicLinkLimitEnabled: true
+        )
+    }
+
+    var betaGroup: BetaGroup {
+        BetaGroup(
+            attributes: nil,
+            id: "1234",
+            relationships: nil,
+            links: ResourceLinks(self: testUrl)
+        )
+    }
+
+    var betaGroupResponseFuture: Future<BetaGroupResponse, Error> {
+        let response =  BetaGroupResponse(
+            data: self.betaGroup,
+            included: nil,
+            links: DocumentLinks(self: URL(fileURLWithPath: "test"))
+        )
+
+        return Future { $0(.success(response)) }
+    }
+
+    var betaGroupsResponseFuture: Future<BetaGroupsResponse, Error> {
+        let response = BetaGroupsResponse(
+            data: [self.betaGroup],
+            included: nil,
+            links: PagedDocumentLinks(first: nil, next: nil, self: self.testUrl),
+            meta: nil
+        )
+
+        return Future { $0(.success(response)) }
+    }
+
+    func testSuccess() throws {
+        let successRequestor = TwoEndpointTestRequestor(
```

Comment:
`TwoEndpointTestRequestor ` is a nice one! but what if there are more than two requests in an Operation 🤔

Reply:
Then we'll need to add another requestor type... It's not my favorite solution. But I think it's what we have unless the SDK changes